docs/tracing.md

```diff
@@ -16,11 +16,12 @@ framework = AgentFramework("openai")
 setup_tracing(framework)
 
 agent = AnyAgent.create(
+        framework,
         main_agent=AgentConfig(
```

Comment:
This is also wrong:

```suggestion
        ¡AgentConfig(
```

Now i want to find a way to run this codeblocks in CI 😅